docs/index.rst

```diff
@@ -110,6 +110,28 @@ View the installed Helm releases:
       helm list
     NAME                REVISION    UPDATED                     STATUS      CHART                   NAMESPACE
     my-confluent-oss    1           Tue Jun 12 16:56:39 2018    DEPLOYED    cp-helm-charts-0.1.0    default 
+Persistence
+~~~~~~~~~~~~~~~~~~~
+The ZooKeeper and Kafka cluster are deployed with ``StatefulSets`` that have a ``volumeClaimTemplate`` which will provide the persistent volume for each replica. We can define the size of the volumes by changing ``dataDirSize`` and ``dataLogDirSize`` under ``cp-zookeeper`` and ``size`` under  ``cp-kafka`` in `values.yaml <https://github.com/confluentinc/cp-helm-charts/blob/master/values.yaml>`__
```

Comment:
suggest (see [Style Guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone)):

```

You can define the size of the volumes by changing...
```